### PR TITLE
Update dependencies 2020-05-28

### PR DESCRIPTION
**Pull request by newhoggy**
- Includes ledger specs patch that fixes transaction validation

### CODE REVIEW

File: cardano-api/src/Cardano/Api/ProtocolParameters.hs [hunk 1]
```diff
@@ -81,12 +81,12 @@ import           Cardano.Slotting.Slot (EpochNo, EpochSize (..))
 import qualified Cardano.Ledger.Core as Ledger
 import qualified Cardano.Ledger.Era as Ledger
 import           Ouroboros.Consensus.Shelley.Eras (StandardShelley)
-import           Ouroboros.Consensus.Shelley.Protocol.Crypto (StandardCrypto)
+import           Ouroboros.Consensus.Shelley.Protocol (StandardCrypto)
```
Comment (Jasagredo) [inline cardano-api/src/Cardano/Api/ProtocolParameters.hs:84]: This one is re-exported from Cardano.Ledger.Crypto. Was unified recently. You might want to import that one instead.

Comment (Jasagredo) [reply cardano-api/src/Cardano/Api/ProtocolParameters.hs:84]: Probably it should be moved even further up into cardano-crypto or something like that I guess.

Comment (newhoggy) [reply cardano-api/src/Cardano/Api/ProtocolParameters.hs]: Updated!

**Review by Jimbo4350 — APPROVED** [review]

**Review by Jimbo4350 — APPROVED** [review]

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**